index.js

```diff
@@ -0,0 +1,91 @@
+let slideIndex = 0;
+
+function moveSlide(n) {
+  const slides = document.querySelectorAll('.carousel__item');
+  const totalSlides = slides.length;
+
+  slideIndex += n;
+
+  if (slideIndex >= totalSlides) {
+    slideIndex = 0;
+  } else if (slideIndex < 0) {
+    slideIndex = totalSlides - 1;
+  }
+
+  updateSlidePosition();
+  updateDots();
+}
+
+function currentSlide(n) {
+  slideIndex = n;
+  updateSlidePosition();
+  updateDots();
+}
+
+function updateSlidePosition() {
+  const slides = document.querySelectorAll('.carousel__item');
```

Comment:
Similar al comentario anterior, usa clases `js-`
```suggestion
  const slides = document.querySelectorAll('.js-carousel-item');
```